src/gsi/cplr_gfs_ensmod.f90

```diff
@@ -443,19 +435,25 @@ subroutine move2bundle_(grd3d,sloc,atm_bundle,m_cvars2d,m_cvars3d,iret)
           write(6,'(A)') trim(myname_) // ': For now, GFS requires all MetFields: ps,u,v,(sf,vp)tv,q,oz'
           write(6,'(A)') trim(myname_) // ': but some have not been found. Aborting ... '
           write(6,'(A)') trim(myname_) // ': WARNING!'
-          write(6,'(3A,I5)') trim(myname_) // ': Trouble reading ensemble file : ', trim(filename), ', IRET = ', iret
        endif
        return
     endif
 
+    do m=1,nc2d
+!      convert ps from Pa to cb
+       if(trim(cvars2d(m))=='ps') ps=en_loc3(:,:,m_cvars2d(m))
```

Comment:
Is the removal of the r0_001 factor taken into account somewhere else?

Reply:
Nevermind, I found it in get_gefs_ensperts_dualres

Reply:
Right!  It was moved so that the data could stay in single precision until it got to get_gefs_ensperts_dualres.  Thanks!